tests/test_authenticator.py

```diff
@@ -2975,7 +2975,15 @@ def test_basic_auth_temp_token_returns_existing(self):
                 another_token = another_response.json.get('access_token')
                 assert another_token == token
 
-    def test_basic_auth_temp_token_returns_new_token(self):
+
+    @pytest.mark.parametrize(
+        'delta',
+        [
+            pytest.param(61, id='after_one_minute'),
+            pytest.param(3601, id='after_one_hour')
```

Comment:
Shouldn't the authenticator return an error if trying to retrieve after more than one hour? Or does changing the `frozen_time` reset things in a way that I'm not following?